core/templates/pages/login-page/login-page.component.ts

```diff
@@ -76,14 +78,28 @@ export class LoginPageComponent implements OnInit {
         return;
       }
 
-      try {
-        await this.authService.signInWithRedirectAsync();
-      // We use unknown type because we are unsure of the type of error
-      // that was thrown. Since the catch block cannot identify the
-      // specific type of error, we are unable to further optimise the
-      // code by introducing more types of errors.
-      } catch (error: unknown) {
-        this.onSignInError(error as firebase.auth.Error);
+      if (
+        this.platformFeatureService.status.SigninWithPopUp.isEnabled
+      ) {
+        try {
+          await this.authService.signInWithPopupAsync();
```

Comment:
@kshitij01042002 So what do you do with the credentials after you get them?

Reply:
For now, I have just console logged the credentials.